packages/types/src/codec/Linkage.ts

```diff
@@ -35,8 +35,8 @@ export default class Linkage<T extends Codec> extends Struct {
 export class LinkageResult extends Tuple {
   public constructor ([TypeKey, keys]: TypeWithValues, [TypeValue, values]: TypeWithValues) {
     super({
-      Keys: Vector.with(TypeKey),
-      Values: Vector.with(TypeValue)
+      Keys: Vec.with(TypeKey),
```

Comment:
This could be moved out as well.

Reply:
Not really, since it needs the Types on construction. (So really it fits at the base)